### PR TITLE
DEV 3107 uwsgi config changes

### PR DESCRIPTION
**Pull request by x110dc**
Please fill out each section even if it's just with "N/A"

#### What's this PR do?

- moves the uwsgi configuration from environment variables in the Dockerfile to an INI-style config file
- installs the uwsgi monitoring app uwsgitop
- upgrades uwsgi to the latest release
- adds new uwsgi configuration options: stop logging requests (except 4xx and 5xx); log details about harikari

#### Why are we doing this? How does it help us?

- it seems clearer to have all the uwsgi configuration in one file in the same format you usually find advise about how to configure it
- this also simplifies the Dockerfile
- uwsgitop may help us diagnose why workers are being killed 
- logging every request pollutes the logs with information that isn't very valuable; this makes the logs easier to read

#### Are there detailed, specific, step-by-step testing instructions in the associated ticket?

Not yet.

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

N/A

#### Have automated unit tests been added? If not, why?

N/A

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

N/A

#### Has this been documented? If so, where?

N/A

#### What are the relevant tickets? Add a link to any relevant ones.

https://news-revenue-hub.atlassian.net/browse/DEV-3107

#### Do any changes need to be made before deployment to production (adding environment variables, for example)? If so, open a ticket and link it to the ticket for this PR and list it here:

No.

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

N/A

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -18,4 +18,4 @@ fi
 
 /bin/sh ./google_sa.sh
 
-exec uwsgi --http=0.0.0.0:$PORT --show-config
+exec uwsgi --ini ./uwsgi.conf --http=0.0.0.0:$PORT --show-config
```
Comment (benjaminEwhite) [inline docker-entrypoint.sh:21]: Looks like `--show-config` here is duplicative of `show-config = true` in uwsgi.conf.

File: pyproject.toml [hunk 2]
```diff
@@ -81,6 +81,7 @@ mailchimp-marketing = "^3.0.80"
 google-cloud-secret-manager = "^2.16.0"
 django-request-id = "^1.0.0"
 pydantic = "^2.2.0"
+uwsgitop = "^0.11"
```
Comment (benjaminEwhite) [inline pyproject.toml:84]: AFAICT, we don't start uwsgitop in the container. Is the plan to do that in a follow up release, or is that something that will be done manually from terminal?

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]
Left one small suggestion and a question about starting uwsgitop.

File: docker-entrypoint.sh (hunk 1 above)
Comment (x110dc) [reply docker-entrypoint.sh:21]: Good catch. I'll remove here.

Comment (x110dc) [reply docker-entrypoint.sh]: https://github.com/newsrevenuehub/rev-engine/commit/eb275ef41

**Review by benjaminEwhite — APPROVED** [review]

**Pull request merged**